scripts/run_frontend_tests.py

```diff
@@ -54,10 +61,35 @@
 )
 
 
+def run_dtslint_type_tests():
+    """Runs the dtslint type tests in typings/tests."""
+    python_utils.PRINT('Running dtslint type tests.')
+
+    cmd = ['./node_modules/dtslint/bin/index.js', DTSLINT_TYPE_TESTS]
+    task = subprocess.Popen(cmd, stdout=subprocess.PIPE)
+    output_lines = []
+    # Reads and prints realtime output from the subprocess until it terminates.
+    while True:
+        line = task.stdout.readline()
+        # No more output from the subprocess, and the subprocess has ended.
+        if len(line) == 0 and task.poll() is not None:
+            break
+        if line:
+            python_utils.PRINT(line, end='')
+            output_lines.append(line)
+    python_utils.PRINT('Done!')
+    if task.returncode:
+        sys.exit('The dtslint (type tests) failed.')
+
+
 def main(args=None):
     """Runs the frontend tests."""
     parsed_args = _PARSER.parse_args(args=args)
 
+    run_dtslint_type_tests()
+    if parsed_args.dtslint_only:
```

Comment:
Do the rest of the tests take significantly longer? If not, we might not need this arg.

Reply:
The rest of the tests take me anywhere from 3-5 minutes, but the dtslint tests only take a couple seconds.